utils/build/docker/java/spring-boot-native.Dockerfile

```diff
@@ -1,17 +1,22 @@
+ARG TRACER_IMAGE=agent_local
 
-FROM eclipse-temurin:8 as agent
+#TODO RMM: The images will be in dd-trace-java repository. Now for tests purposes we are using system-tests repository
```

Comment:
#TODO RMM: The images will be in dd-trace-java repository. Now for tests purposes we are using system-tests repository